content/en/docs/release-oversight/troubleshooting-payload-failures.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Troubleshooting Payload Failures"
+description: A guide to tools and strategies used to diagnose payload failures.
+---
+
+### Problem Statement
+There is a lot of data collected during payload testing and multiple tools to assist in analyzing that data.
+However, the data is not complete and some tools are better at capturing & analyzing particular types or subsets of data than others.
+Additionally, with the number of jobs, job failures and individual tests it can be hard to determine how / where to start.
+There is no step by step guide but we hope to document the benefits of the tools we have available and approaches to help investigate payload failures.
+
+### Description of Tools and Usages
+Typically when analyzing an individual prow run there are currently 3 tools listed at the top of the job as well as additional
+tools to check for larger trends related to tests, errors and events.
+
+- Loki - Grafana logging stack allowing you search logs and narrow time ranges.  This contains intermittent pod log entries that are not present in the must-gather or gather-extra artifacts.  As well as the kubelet journal logs found under gather-extra and others to aid in searching.
+
+
+- KaaS - Representation of the last known state of the cluster.  Can be useful for navigating the cluster via OC or K9S and interrogating namespaces, logs, etc.
+
+
+- PromCIeus - Metrics from the cluster during the test run loaded in Prometheus.
+
+
+- CI Search - https://search.ci.openshift.org/ - If you have found a particular log entry and want to search bugs and other junit runs to find:
+    - How frequently does it occur
+    - Did it start or spike recently
+    - Is there an open bug against it <br><br>
+
+- Sippy - https://sippy.dptools.openshift.org/sippy-ng/ Used to evaluate recent trends for aggregated jobs and individual tests.  You can filter by Release, Job, or Test as well as variants like platform, network, etc.  If you have found a job or test and want to dig deeper to determine how often does it fail, does it fail more ofter for particular variants, has it recently degraded.
+
+
+- Data Studio - https://datastudio.google.com/navigation/reporting The big hammer with big data.  This is our data warehouse.  Like Sippy but at a larger scale we can search, query and dissect job and test results.
+
+- Test Grid - https://testgrid.k8s.io/ (Search for a dashboard/tab: redhat-openshift ...) 
+
+### Artifacts
+The prow job will have a link to the artifacts collected during the run.  Some of the highlights contained in the artifacts are
+
+- The main build.log.txt in the root of the folder
+- Gather-must-gather 
```

Comment:
add a link to this https://github.com/openshift/must-gather which is how that data is gathered.